packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -135,7 +135,13 @@ export const backgroundPositionToCoords = ( value ) => {
 };
 
 function InspectorImagePreview( { label, filename, url: imgUrl } ) {
-	const imgLabel = label || getFilename( imgUrl );
+	let imgLabel = label || getFilename( imgUrl );
+	// getFilename returns 'undefined' as a string.
```

Comment:
Why? Sounds like a bug we should fix.

Reply:
https://github.com/WordPress/gutenberg/pull/60265

Reply:
I just wrote that comment to explain why I wrote the condition that way. The PR above should fix it.